projects/common/window.cpp

```diff
@@ -3,6 +3,11 @@
 #include <cassert>
 #include <fstream>
 
+#if defined(__APPLE__)
+#include <unordered_map>
```

Comment:
Anything actually use from this include?

Reply:
I thought so, but apparently not. BALEETED!